### PR TITLE
Add function to rebuild hg workspace on a new machine

### PR DESCRIPTION
**Pull request by johnoliver**

### CODE REVIEW

File: git-hg/jdk8u/merge.sh [hunk 1]
```diff
@@ -144,6 +183,9 @@ while getopts "ab:irts:T:u" opt; do
             doReset="true"
             doInit="true"
             ;;
+        R)
+            doRebuildLocalRepo="true"
```
Comment (karianna) [inline git-hg/jdk8u/merge.sh]: Better to use `l`?

Comment (johnoliver) [reply git-hg/jdk8u/merge.sh]: ok

**Comment by johnoliver** [comment]
I would wait to make sure this works correctly first

File: git-hg/jdk8u/merge.sh [hunk 2]
```diff
@@ -108,6 +109,44 @@ function updateRepo() {
...
+    #
+    # 3. Set up remotes on $REPO
+    #     Remotes should look as follows:
+    #       upstream: git@github.com:AdoptOpenJDK/openjdk-jdk8u.git (or aarch)
+    #       root:    "$MIRROR/root/"
+    #       origin:  "$MIRROR/root/"
+    #
+
+    # Step 1 Clone mirrors
+    updateMirrors $hgRepo
```
Comment (karianna) [inline git-hg/jdk8u/merge.sh]: Does that clone or update or is that function a clone and/or update?

File: git-hg/jdk8u/merge.sh [hunk 3]
```diff
@@ -108,6 +109,44 @@ function updateRepo() {
...
+
+    # Step 2, Reclone upstream repo
+    rm -rf "$REPO" || true
+    mkdir -p "$REPO"
+    cd "$REPO"
+    git clone $UPSTREAM_GIT_REPO .
+    git checkout master
+
+    # Step 3 Retup remotes
```
Comment (karianna) [inline git-hg/jdk8u/merge.sh]: Retup --> Setup

File: git-hg/jdk8u/merge.sh (hunk 2 above)
Comment (johnoliver) [reply git-hg/jdk8u/merge.sh]: yes if it does not exist it creates it, and updates it

**Pull request merged**